tools/get-boost.sh

```diff
@@ -83,6 +83,7 @@ git submodule update --init --depth 20 --jobs 4 \
     libs/type_index \
     libs/typeof \
     libs/unordered \
+    libs/variant2 \
```

Comment:
Did Beast use variant2 before?

Reply:
No, it looks to be a transitive dependency from boost/system from by json example. I am currently searching for where it's included from.

Reply:
Added to json as a dep 3 days ago, so that would be it.

Reply:
Ok